.github/workflows/netlify-preview.js.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Netlify'
+
+on:
+  pull_request:
+    types: [labeled]
```

Comment:
This event will trigger the action only when we add a label to the PR. Below we execute the job only if that new label matches `preview`.